platformio.ini

```diff
@@ -128,7 +128,7 @@ monitor_speed = 115200
 src_dir=speeduino
 default_envs = megaatmega2560
 ;The following lines are for testing / experimentation only. Comment the line above to try them out
-;default_envs = black_F407VE
+default_envs = black_F407VE
```

Comment:
Can you comment back out this line for the PR? Default environment should remain the 2560